site/en/docs/extensions/reference/userscript/index.md

```diff
@@ -0,0 +1,46 @@
+---
+api: userScript
+---
+
+## Using now
```

Comment:
I really wish this section was an aside 🤔

Reply:
Generally you only need one way to draw attention to something. This already has the advantage of being close to the top.